reference/test/preconditioner/jacobi_kernels.cpp

```diff
@@ -1108,4 +1110,156 @@ TYPED_TEST(Jacobi, ConvertsEmptyToDense)
 }
 
 
+TYPED_TEST(Jacobi, ScalarL1Diag)
+{
+    /* example matrix:
+        1   1
+        2   2
+       -5       1
+        1      -1
+     */
+    using Mtx = typename TestFixture::Mtx;
+    using index_type = typename TestFixture::index_type;
+    using value_type = typename TestFixture::value_type;
+    auto mtx = Mtx::create(this->exec, gko::dim<2>{4}, 8);
```

Comment:
```suggestion
    auto mtx = Mtx::create(this->exec, gko::dim<2>{4, 4}, 8);
```
IMO the single parameter `dim` constructor is a bit confusing.